src/ResourceManager/Compute/Commands.Compute.Test/packages.config

```diff
@@ -19,7 +18,9 @@
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.18.206251556" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime" version="1.4.1" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="0.11.0" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="0.9.3" targetFramework="net45" />
```

Comment:
@hyonholee Is there any reason this needs to be downgraded from 0.11.0 to 0.9.3?